FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Utils/StringUtils.swift

```diff
@@ -0,0 +1,58 @@
+import FirebaseAuth
+import SwiftUI
+
+let kKeyNotFound = "Key not found"
+
+let kUsersNotFoundError = "UserNotFoundError"
+let kEmailAlreadyInUseError = "EmailAlreadyInUseError"
+let kInvalidEmailError = "InvalidEmailError"
+let kWeakPasswordError = "WeakPasswordError"
+let kSignUpTooManyTimesError = "SignUpTooManyTimesError"
+let kWrongPasswordError = "WrongPasswordError"
+let kAccountDisabledError = "AccountDisabledError"
+let kEmailsDoNotMatchError = "EmailsDoNotMatchError"
+let kUnknownError = "UnknownError"
+
+public class StringUtils {
```

Comment:
Let's see if using SwiftUI's string localization is a more canonical way to support this.

See https://developer.apple.com/documentation/xcode/localizing-and-varying-text-with-a-string-catalog